README.md

```diff
@@ -1,2 +1,109 @@
+
+## Problem areas
+
+These are the problem areas that we identified, and are described in further detail later in the document:
+
+1. Data access
+2. Design components
+3. Out-of-band decisions
+4. Participant accounts
+
+### Data access
+
+#### The problem
+
+Data essential to the Bamboo experience is tied into the CGA environment and not exposed via a web API or other such mechanism. This data includes things like:
+
+- Issue guides
+- Options
+- Actions
+- Drawbacks
+
+#### The solution
+
+NewAperio will build a proxy server application that will connect directly to CGA’s database instance via a read-only connection and expose the required data over a well-defined GraphQL web API.
+
+We discussed other options with the CGA team that would involve them making non-neglible changes to the exising legacy codebase:
+
+- Experimenting with [Lift’s REST helpers][lift-rest]
+- Building a bespoke [JSON:API] service
+
+After evaluating the tradeoffs, both teams decided that the solution we chose would be best--it ensures we have access to the data Bamboo needs and requires no modification of the legacy app.
+
+### Design components
+
+#### The problem
+
+Some of the content included in the [prototype designs] are not available in the CGA application. This includes:
+
+- Issue guide cover images
+- Issue guide categories
+
+#### The solution
+
+After consulting with our senior designer, we feel like we can build Bamboo without this content and not unduly compromise the user experience.
+
+This content, though not essential, would be nice to have and we’ve identified a couple of potential pathways for getting access to it:
+
+- The CGA team could add fields to the legacy app to allow issue guide authors to specify an image and category
+- The [EVE website][eve], a WordPress app, has the content we need and there are plugins for WordPress that can provide a drop-in web API. The difficult part of this approach, however, is linking up the WordPress data with the correct CGA issue guides.
+
+Both of these options require work from the CGA team but not much work on our part, so we will defer to CGA on this decision.
+
+### Out-of-band decisions
+
+#### The problem
+
+In the current CGA app, participant decisions, e.g. chosen options, actions, evaluations, are recorded as part of an event stream during the forum experience. Bamboo records these decisions outside of the forum experience and CGA has not yet decided how they will be reconciled with the the event stream paradigm.
+
+### The solution
+
+NewAperio will store user decisions as part of the Bamboo application. When a complete set of decisions has been recorded, Bamboo will send a webhook event to CGA with information about how to retrieve the records. This decouples Bamboo’s implementation from the CGA app, leaving the CGA team to design their solution without restrictions.
```

Comment:
Since it's inside the same VPC in AWS, we could also use something like SQS so the CGA app can just read it at its leisure instead of us forcing it to act on an HTTP request. Whichever they prefer, not sure it matters to us.